Dfe.Data.SearchPrototype/Web/Views/Home/Index.cshtml

```diff
@@ -68,30 +68,32 @@
             <div class="govuk-grid-column-two-thirds" id="results">
                 <h3 class="govuk-heading-m" id="search-results-count">@Model.SearchResultsCount @(Model.HasMoreThanOneResult ? "Results" : "Result")</h3>
                 <hr />
-                <ul style="list-style: none">
+                <ul id="establishment-search-results" style="list-style: none">
                     @foreach (var searchItem in Model.SearchItems!)
                     {
-                        <li id="name-@searchItem.Urn">
```

Comment:
Going to have to fix the conflict of moving to shared components on this PR and then merge outwards into dependant PRs